tests/bluetooth/shell/src/main.c

```diff
@@ -23,28 +23,47 @@
 
 #include <shell/shell.h>
 
+#include <bluetooth/hci.h>
+#include <bluetooth/bluetooth.h>
 #include <bluetooth/services/hrs.h>
 
 #define DEVICE_NAME CONFIG_BT_DEVICE_NAME
 
 #if defined(CONFIG_BT_GATT_HRS)
 static bool hrs_simulate;
 
+static const struct bt_data ad[] = {
+	BT_DATA_BYTES(BT_DATA_FLAGS, (BT_LE_AD_GENERAL | BT_LE_AD_NO_BREDR)),
+	BT_DATA_BYTES(BT_DATA_UUID16_ALL, 0x0d, 0x18, 0x0f, 0x18, 0x0a, 0x18),
+};
+
 static int cmd_hrs_simulate(const struct shell *shell,
 			    size_t argc, char *argv[])
 {
-	if (!strcmp(argv[1], "on")) {
-		static bool hrs_registered;
+	static bool hrs_registered;
+	int err;
 
+	if (!strcmp(argv[1], "on")) {
 		if (!hrs_registered) {
 			shell_print(shell, "Registering HRS Service");
 			hrs_registered = true;
+			err = bt_le_adv_start(BT_LE_ADV_CONN_NAME, ad,
+					      ARRAY_SIZE(ad), NULL, 0);
+			if (err) {
+				shell_error(shell, "Advertising failed to start"
+					    " (err %d)\n", err);
+				return -ENOEXEC;
+			}
```

Comment:
minor: please add an empty line after the {

Reply:
Done